src/cobra/test/test_util/test_solver.py

```diff
@@ -38,13 +48,17 @@ def test_choose_solver(model):
             su.choose_solver(model, qp=True)
 
 
-def test_linear_reaction_coefficients(model):
+def test_linear_reaction_coefficients(model: "Model") -> None:
+    """Test if linear coefficients are identifiable in objective."""
     coefficients = su.linear_reaction_coefficients(model)
     assert coefficients == {model.reactions.Biomass_Ecoli_core: 1}
 
 
 @pytest.mark.parametrize("solver", optlang_solvers)
-def test_fail_non_linear_reaction_coefficients(model, solver):
+def test_fail_non_linear_reaction_coefficients(
+    model: "Model", solver: List[str]
+) -> Optional[ValueError]:
```

Comment:
In this test it'd be nicer to only set GLPK and expect the `ValueError` always.

```suggestion
) -> None:
```

Reply:
I have updated the test.